{"name": "Тест невозможности авторизации зарегистрированного курьера при вводе некорректных данных (статус ответа 404)", "status": "passed", "steps": [{"name": "Авторизация курьера", "status": "passed", "parameters": [{"name": "login", "value": "'incorrect_data'"}, {"name": "password", "value": "'dxhmghdmsh'"}], "start": 1734893590681, "stop": 1734893590922}], "parameters": [{"name": "field_name", "value": "'login'"}], "start": 1734893590681, "stop": 1734893590923, "uuid": "6e1ae65d-26e9-4a81-b448-a7e014388e5d", "historyId": "622bbe5c82f55c413bbf6fa850ea6e8b", "testCaseId": "62de0a1455e4348f76fb13991ea19ed5", "fullName": "tests.couriers.test_login_courier.TestLoginCourier#test_login_courier_with_incorrect_login_or_password", "labels": [{"name": "parentSuite", "value": "tests.couriers"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-TPCCV60"}, {"name": "thread", "value": "12028-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.couriers.test_login_courier"}]}